encryption-decryption/src/com/nbugaenco/encryptdecrypt/ShiftEncoder.java

```diff
@@ -0,0 +1,43 @@
+package com.nbugaenco.encryptdecrypt;
+
+public class ShiftEncoder extends Encoder {
+
+    public ShiftEncoder() {
+    }
+
+    @Override
+    public String encode() {
+        return prompt.codePoints()
+                .map(c -> {
+                    if (c >= 65 && c <= 90) {
```

Comment:
Try to avoid lambda which has more than 2 lines (_keep code human readable_) you change it to something like that:

```java

private Integer encode(final Integer c){
 if (c >= 65 && c <= 90) {
   c = (c + key > 90) ? (c + key) - 26 : c + key;
 } else if (c >= 97 && c <= 122) {
   c = (c + key > 122) ? (c + key) - 26 : c + key;
}
return c
}

@Override
    public String encode() {
        return prompt.codePoints()
                .map(this::decode)
                .collect(...).toString();
```